src/test/java/itm/oss/splitter/AppTest.java

```diff
@@ -0,0 +1,99 @@
+package itm.oss.splitter;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.math.BigDecimal;
+
+import org.junit.jupiter.api.DisplayName;
+import java.io.ByteArrayInputStream;
+
+public class AppTest {
+
+    // JUnit 5 기본 구조
+    @Test
+    void testApp_Placeholder() {
+        assertTrue(true); 
+    }
+
+    private final InputStream originalIn = System.in;
+    private final PrintStream originalOut = System.out;
+
+    private App createAppWithSimulatedInput(String input) {
+        System.setIn(new ByteArrayInputStream(input.getBytes()));
+        return new App();
+    }
+
+    @Test
+    @DisplayName("Edge Case: readRequiredLine rejects empty input")
+    void testReadRequiredLine_EmptyInput_ShowsError(){
+        ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(outContent));System.setOut(new PrintStream(outContent));
```

Comment:
There are two consecutive calls to `System.setOut(new PrintStream(outContent));` on the same line.  
The second one immediately overrides the first, so it’s redundant.  

So I recommend remove redundancy for better readability.

```suggestion
        System.setOut(new PrintStream(outContent));

Reply:
Thanks for pointing that out redundant problem.
That was a clear issue we must change it.
I've committed the suggested change.
Please approve if it looks good!

Reply:
I've confirmed that the redundancyn is deleted.
LGTM^^